src/main.js

```diff
@@ -81,21 +81,29 @@ function displayRandomHomePage() {
   displayRandomTitle();
   displayRandomTagline1();
   displayRandomTagline2();
+  currentCover = new Cover(htmlImage.src, htmlTitle.innerText, htmlTagline1.innerText, htmlTagline2.innerText);
+  console.log(currentCover);
```

Comment:
Remove all `console.log` and commented out code when committing to main, unless it is specifically there for another user to see (this should be pretty rare).